integration-test/src/test/java/org/jboss/pnc/integration_new/endpoint/RestTest.java

```diff
@@ -0,0 +1,283 @@
+/**
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014-2020 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.integration_new.endpoint;
+
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import org.assertj.core.api.Assertions;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.junit.InSequence;
+import org.jboss.pnc.AbstractTest;
+import org.jboss.pnc.common.json.JsonUtils;
+import org.jboss.pnc.common.util.IoUtils;
+import org.jboss.pnc.dto.Product;
+import org.jboss.pnc.dto.Project;
+import org.jboss.pnc.integration.matchers.JsonMatcher;
+import org.jboss.pnc.integration_new.setup.Deployments;
+import org.jboss.pnc.test.category.ContainerTest;
+import org.jboss.shrinkwrap.api.spec.EnterpriseArchive;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.lang.invoke.MethodHandles;
+
+import static io.restassured.RestAssured.given;
+import static org.jboss.pnc.integration.env.IntegrationTestEnv.getHttpPort;
+
+/**
+ * @author <a href="mailto:dbrazdil@redhat.com">Dominik Brazdil</a>
+ *
+ * @see org.jboss.pnc.demo.data.DatabaseDataInitializer
+ */
+@RunAsClient
+@RunWith(Arquillian.class)
+@Category(ContainerTest.class)
+public class RestTest extends AbstractTest {
+
+    public static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+    private static final String USER_REST_ENDPOINT = "/pnc-rest-new/rest-new/users/";
+    private static final String PRODUCT_REST_ENDPOINT = "/pnc-rest-new/rest-new/products/";
+    private static final String PRODUCT_VERSION_REST_ENDPOINT = "/pnc-rest-new/rest-new/product-versions/";
+    private static final String PROJECT_REST_ENDPOINT = "/pnc-rest-new/rest-new/projects/";
```

Comment:
extract the common API path to separate constant please (it will change in the future)